TAF_EasyRest/TestFramework/Pages/Client/RestaurantMenuPage.cs

```diff
@@ -30,5 +21,14 @@ public CartPageComponent ClickShowCartButton()
             _showCartButton.Click();
             return new CartPageComponent(driver);
         }
+
+        public RestaurantMenuPage AddTheFistDishToTheCart()
+        {
+            var _menuItem = _menuItemSummaryCards.ElementAt(0);
+            
+            var _addToCartButton = _menuItem.FindElement(By.XPath(".//button[contains(@class,'addButton')]"));
```

Comment:
I believe you've forgot to click on the button :)

Reply:
done)